develop/tutorials/articles/160-import-export-and-staging/04-creating-staged-models-manually.markdown

```diff
@@ -0,0 +1,170 @@
+# Creating Staged Models Manually
+
+There are times when using
+[Service Builder to generate your staged models](/develop/tutorials/-/knowledge_base/7-0/#important-attributes-in-staging)
+is not practical. In these cases, you should create your staged models manually.
+Make sure to read the
+[Adapting Your Business Logic to Build Staged Models](/develop/tutorials/-/knowledge_base/7-0/understanding-staged-models#adapting-your-business-logic-to-build-staged-models)
+section to determine if creating staged models manually is beneficial for your
+use case.
+
+In this tutorial, you'll explore the Asset Link framework (used for
+[Relating Assets](/discover/portal/-/knowledge_base/7-0/defining-content-relationships)),
+which is an internal @product@ framework that leverages the
```

Comment:
@product@ is not resolved for me in github's UI, but maybe it is just an issue with github